src/components/layout/MobileNav.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import Link from "next/link";
+import clsx from "clsx";
+
+import { menus } from "./helper";
```

Comment:
Normally we do not use spaces between imports unless if there is some logical reason.